doc/api/api_changes/2015-11-12-DDH.rst

```diff
@@ -0,0 +1,16 @@
+Modified the 3D quiver function in mpl_toolkits.mplot3d.axes3d.py to have the dafaults: normalize=False, pivot='tail'
+```````````````````````````
+Matplotlib has both a 2D and a 3D "quiver" function. These changes affect only the 3D function and make the default behavior of the 3D function match 2D version. There are two main changes:
+
+1) The 3D quiver function previously normalized the arrows to be the same length, which makes it unusable for situations where the arrows should be different lengths and does not match the behavior of the 2D function. This normalization behavior is now controlled with the "normalize" keyword, which defaults to False. 
+
+2) The "pivot" keyword now defaults to "tail" insteaf of "tip". This was done in order to match the default behavior of the 2D quiver function.
+
+To obtain the previous behavior with the 3D quiver function, one can call the function with:
```

Comment:
To work with rst the code examples should look like

```rst
last text line ::

   code example
```

Both the blank line and the 3 (!) spaces of indent matter